src/components/common/YourCompany/StartDate.vue

```diff
@@ -144,21 +158,15 @@ export default class StartDate extends Mixins(CommonMixin, DateMixin) {
     return this.dateToYyyyMmDd(this.startDateMax)
   }
 
-  /** The recognition date or business start date-time string. */
-  protected get recognitionDateTime (): string {
-    if (this.isBenIaCorrectionFiling) {
-      if (this.getCorrectedFilingDate) {
-        return this.yyyyMmDdToPacificDate(this.getCorrectedFilingDate, true)
-      }
-    }
+  /** The business date or business start date string. */
+  protected get businessStartDate (): string {
     if (
       this.isFirmCorrectionFiling ||
-      this.isAlterationFiling ||
```

Comment:
Does this component (or YourCompany component) still work for alteration filings?

Reply:
I am still running the old code so just ask if you want a screenshot of any Edit UI filing type.

Reply:
Yes YourCompany component still works for alteration filings, they take the recognitionDateTime instead of businessStartDate

Reply:
Perfect.
Do they display date+time or just date atm?

Reply:
I changed it back to date time on my local machine, but on GH its just the date. Right now I am having issues with saving and getting the dates it sounds similar to the save issues that you are having with the NOB. I am wondering if the issue you are getting is a cloneDeep issue? That is what I am getting right now

Reply:
I am considering merging my PR ASAP and then you can rebase and see if things are fixed. Thoughts?